tests/run.sh

```diff
@@ -91,31 +90,13 @@ AWESOME_CLIENT="$root_dir/utils/awesome-client"
 
 # Start awesome.
 start_awesome() {
-    (
-        export DISPLAY="$D"
-        cd $root_dir/build
-        # Setup xrdb, for awesome's xresources backend / queries.
-        echo "Xft.dpi: 96" | xrdb
-        "$AWESOME" -c "$RC_FILE" $AWESOME_OPTIONS > $awesome_log 2>&1 &
-    )
-
-    # Get PID of awesome.
-    awesome_pid=
-    max_wait=30
-    while true; do
-        awesome_pid="$(pgrep -nf "awesome -c $RC_FILE" || true)"
-        if [ -n "$awesome_pid" ]; then
-            break;
-        fi
-        max_wait=$(expr $max_wait - 1)
-        if [ "$max_wait" -lt 0 ]; then
-            echo "Error: Failed to start awesome (-c $RC_FILE)!"
-            echo "Log:"
-            cat "$awesome_log"
-            exit 1
-        fi
-        sleep 0.1
-    done
+    export DISPLAY="$D"
+    cd $root_dir/build
+    # Setup xrdb, for awesome's xresources backend / queries.
+    echo "Xft.dpi: 96" | DISPLAY="$D" xrdb
```

Comment:
I *guess* that this will otherwise result in a warning that the grep at the end of this file would consider an error. Just a guess. Anyway, I didn't add this and shouldn't mess with it in this unrelated change.

A quick test results in:
    ...chon/projects/awesome/build/lib/beautiful/xresources.lua:67: W: Failed to get xrdb value 'Xft.dpi'.

The `grep` wouldn't trigger on this.